frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -484,6 +484,266 @@ describe('NewRunParametersV2', () => {
     screen.getByDisplayValue('"A":1,"B":2');
   });
 
+  it('test Start button is enabled for valid default integer input', () => {
+    const setIsValidInputSpy = jest.fn();
+    const props = {
+      titleMessage: 'default Title',
+      pipelineRoot: 'defalut pipelineRoot',
+      specParameters: {
+        intParam: {
+          parameterType: ParameterType_ParameterTypeEnum.NUMBER_INTEGER,
+          defaultValue: 123,
+        },
+      },
+      handlePipelineRootChange: jest.fn(),
+      handleParameterChange: jest.fn(),
+      setIsValidInput: setIsValidInputSpy,
+    };
+    render(<NewRunParametersV2 {...props} />);
+
+    expect(setIsValidInputSpy).toHaveBeenCalledTimes(1);
+    expect(setIsValidInputSpy).toHaveBeenLastCalledWith(true);
+    screen.getByDisplayValue('123');
+  });
+
+  it('test error message for invalid integer input', () => {
```

Comment:
We have `test ..` as the prefix of each test name, it is usually not needed in the test name because everything is test when we are using `it('', () => {})`. We can improve naming with the following 
```
it('show error message for invalid integer input', () => {})
```

Same for other test names.